src/CONST.js

```diff
@@ -5,15 +5,18 @@ const CONST = {
     PDF_VIEWER_URL: '/pdf/web/viewer.html',
     EXPENSIFY_ICON_URL: `${CLOUDFRONT_URL}/images/favicon-2019.png`,
     UPWORK_URL: 'https://www.upwork.com/ab/jobs/search/?q=Expensify%20React%20Native&user_location_match=2',
-    REPORT: {
-        SINGLE_USER_DM: 'singleUserDM',
-        GROUP_USERS_DM: 'groupUsersDM',
+    OPTION_TYPE: {
+        REPORT: 'report',
+        PERSONAL_DETAIL: 'personalDetail',
     },
     MODAL: {
         MODAL_TYPE: {
             CENTERED: 'centered',
         },
     },
+    LOGIN: {
+        EXPENSIFY_SMS_SUFFIX: '@expensify.sms'
```

Comment:
This should exist in `expensify-common`

https://github.com/Expensify/expensify-common/blob/d1bfacca6fb92f7e89a2a9cdf0863fad6481c0f2/lib/CONST.jsx#L488